javascript/generate/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "projectName": "lusid-sdk",
   "usePromises": true,
-  "packageVersion": "0.10.682"
+  "packageVersion": "0.11.2849"
```

Comment:
Note that this gets overriden everytime the SDK is generated and doesn't have any meaning.